### ISSUE TITLE
Provide CI-friendly log level that only prints actual errors/warnings

### ISSUE THREAD

**Issue by phoerious** [linuxdeploy/linuxdeploy#119]
linuxdeploy is quite verbose at the "error-only" verbosity level printing messages like

```
  appimagetool, continuous build (commit 64321b7), build 2111 built on 2019-11-23 22:20:53 UTC
  Using architecture x86_64
  Deleting pre-existing .DirIcon
  Creating .DirIcon symlink based on information from desktop file
  Generating squashfs...
```

```
  Embedding ELF...
  Marking the AppImage as executable...
  Embedding MD5 digest
  zsyncmake is available and updateinformation is provided, hence generating zsync file
  Success
```

and worst of all

```
  Please consider submitting your AppImage to AppImageHub, the crowd-sourced
  central directory of available AppImages, by opening a pull request
```

to stderr. While stderr is generally the correct stream for status messages, I would appreciate a log level where only real warnings and errors are shown, so I can configure my CI system to treat stderr output as actual errors.

Furthermore, linuxdeploy isn't very consistent about which output stream it uses for status messages, since, e.g., all the squashfs output goes to stdout, which first of all is generally wrong and secondly it isn't silenced at verbosity level 1 either despite neither being a warning nor an error.

**Comment by TheAssassin** [comment]
The messages you mention are generated by the AppImage plugin. linuxdeploy just forwards the logs it receives from that plugin. The plugin itself uses appimagetool internally, which is responsible for the actual messages.

Since linuxdeploy is plugin based, one cannot simply control verbosity of tools centrally.

The linuxdeploy logging system should consistently use `stdout` for all logging, see https://github.com/linuxdeploy/linuxdeploy/blob/master/include/linuxdeploy/core/log.h.

Your messages miss the `[linuxdeploy/appimage]` prefix. If that has been in the log, you should *never* omit that.

**Comment by phoerious** [comment]
Yeah, I noticed that, although I edited my post, because it's not just the appimagetool creating info log messages at an error debug level.

> Since linuxdeploy is plugin based, one cannot simply control verbosity of tools centrally.

You could require all plugins to accept a --verbosity flag.

**Comment by TheAssassin** [comment]
I don't want to bloat the amount of CLI flags, to be honest. The ones we have in place already is pretty much all I'd require. Perhaps an environment variable like `VERBOSITY` could be established? We use env vars to allow users to influence plugins' behavior. I think the Qt plugin already provides a `$DEBUG`, which could be replaced/extended by a standalone `VERBOSITY` or `LOGLEVEL` flag. 

Of course we can't guarantee for plugins to implement that properly, but then we'd have a standard at least, and you could go file issues with the plugins.
